components/common/confirmPurchaseModal/ConfirmPurchaseModal.vue

```diff
@@ -117,6 +120,10 @@ const totalWithRoyalties = computed(
   () => totalNFTsPrice.value + totalRoyalties.value,
 )
 
+const supportFee = computed(() =>
+  isRemark.value ? totalNFTsPrice.value * SUPPORT_FEE_PERCENT : 0,
+)
+
```

Comment:
no support on other chains?

Reply:
from what I saw the support fee it's only present for Remark, please correct me if I'm wrong

Reply:
yep. when buying an nft support is only present in `execBuyRmrk`

@roiLeo @vikiival , should be when buying on any chain. right?

Reply:
in pipe #5130